### DISCUSSION TITLE
Does "Total Commits (2023)" refer to one year's Commits for 2023, or one year's Commits for the past year (22.10-23.10)?

### DISCUSSION THREAD

**Discussion by ghost** [anuraghazra/github-readme-stats#3356] [Q&A] [answered]
Does "Total Commits (2023)" refer to one year's Commits for 2023, or one year's Commits for the past year (22.10-23.10)?

**Comment by qwerty541** [comment, marked as answer]
Hey, @zly5! "Total Commits (2023)" refers to commits that was made in the last 365 days.
